.gitlab-ci.yml

```diff
@@ -11,3 +11,5 @@ push:
   stage: push
   script:
     - docker push gpii/exekube:0.4.0-google
+  only:
+    - master@gpii-ops/gpii-infra
```

Comment:
shouldn't this say `master@gpii-ops/exekube`?

Reply:
Oops.